types/field_type_test.go

```diff
@@ -199,6 +203,25 @@ func (s *testFieldTypeSuite) TestDefaultTypeForValue(c *C) {
 	}
 }
 
+// BenchmarkDefaultTypeForValue
+// run with: GO111MODULE=on go test -bench=BenchmarkDefaultTypeForValue
+var r uint64 = 0xdeadbeef
+
+// Pseudo-random number generator adapted from
+// https://github.com/dgryski/trifles/blob/master/fastrand/fastrand.go
+func Rand() int64 {
+	r ^= r >> 12 // a
+	r ^= r << 25 // b
+	r ^= r >> 27 // c
+	return int64(r * 2685821657736338717)
+}
+func BenchmarkDefaultTypeForValue(b *testing.B) {
```

Comment:
Maybe we should add an empty line here.
Question:  do we need to export Rand here?

Reply:
probably not. So you mean I should make the name Rand start with small letter like rand?

Below I was told to "Extract these newly added vars and function to a new file types/benchmark_test.go". Perhaps I should still keep it starting with capital letter?

Reply:
Oh, I see the benchmark test is moved there as well, perhaps I can make it start with small letter.